### ISSUE TITLE
Improve `ockam identity create` with `--output json` flag

### ISSUE THREAD

**Issue by nazmulidris** [build-trust/ockam#6131]
# Current behavior

If you run `ockam identity create --output json` you get output similar to this:

![Image](https://github.com/build-trust/ockam/assets/2966499/e8052643-1a1e-4369-b8f6-0900d68d2af8)

A spinner is output to the `stderr` so it doesn't break scripts.

# Desired behavior

The spinner shouldn't appear when running with this flag.

Please take a look at this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/identity/create.rs#L48) to make changes.

---

<i>We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).</i>

**Comment by axd99** [comment]
Hi @nazmulidris,

Could you assign this issue to me?

**Comment by nazmulidris** [comment]
@axd99 That's awesome, this is all yours. Please let us know if you have any questions as you explore. You can also ask questions on the contributors discord https://discord.gg/RAbjRr3kds

**Comment by axd99** [comment]
Thank you @nazmulidris .

And thanks for the invite to the contributors discord.

 I've tried to access the contributors discord on the link above, but I get an error message 'Unable to accept invite'. 
 
 I have raised a ticket with the discord help desk, and they have come back to me (very quickly I must add) confirming that the link is valid but that my IP address must be banned (somehow) and that I should ask the moderator 'to clear all/some bans in Server Settings > Bans'.
 
I should then restart my client and try again.

Or it could also be (this is still from the help desk) with my discord tag being in a black listed (in a moderator bot?). 

I have joined Discord recently and have has very limited activity.
Any chance someone could have a look on your side?
